src/edge-core-index.js

```diff
@@ -686,6 +688,7 @@ export type EdgeCurrencyEngine = {
   getBalance(options: any): string,
   getNumTransactions(options: any): number,
   getTransactions(options: any): Promise<Array<EdgeTransaction>>,
+  getTxids(): Array<string>,
```

Comment:
Can you make this optional as not all plugins have this call? Or does the remaining code absolutely need it? Ideally this is an optimization.